docs/sphinx/source/whatsnew/v0.9.0.rst

```diff
@@ -180,6 +180,8 @@ Bug fixes
   when data time step is not one minute. Error was introduced in v0.8.1.
   (:issue:`1241`, :pull:`1242`)
 * Corrected :py:func:`~pvlib.solarposition.spa_python` to avoid a future warning. (:pull:`1256`)
+* Changed deprecated use of ``.astype()`` to ``.view()`` in :py:mod:`~pvlib.solarposition`.
+  (:issue:`1261`, :pull:`1262`)
```

Comment:
Could merge this with the entry above it, nbd though

Reply:
Merged as suggested.